features/partnerhistorie/peildatum-periode-filtering.feature

```diff
@@ -0,0 +1,258 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Selecteer het/de correcte (historische) huwelijken geregistreerde partnerschappen.
+
+Rule: Bij vragen van partnerhistorie op peildatum wordt de partner geleverd waarbij de Datum huwelijkssluiting/aangaan geregistreerd partnerschap op of voor de peildatum ligt én de eventuele Datum ontbinding huwelijk/geregistreerd partnerschap na de peildatum.
+
+    Abstract Scenario: Peildatum in actuele huwelijk/partnerschap
```

Comment:
Waarom is deze scenario abstract gemaakt? De scenario heeft geen voorbeelden.

Verder denk ik dat de titel wat smarter moet worden gemaakt. Als ik naar de stappen kijk, dan denk ik dat het volgende wordt beschreven met de scenario:

datum huwelijkssluiting/aangaan geregistreerd partnerschap van een actuele huwelijk ligt voor de peildatum

Reply:
@MelvLee moet een scenariotitel al zo technisch specifiek zijn?

Reply:
Ik zie een scenario als een hoofdstuk in een boek en de titel van de scenario komt dan overeen met de titel van de hoofdstuk. Persoonlijk kan ik niet zeggen of de titel die ik heb voorgesteld te technisch is.

Naast deze scenario zou ik nog een andere scenario hebben opgenomen: datum huwelijk/aangaan geregistreerd partnerschap van een actuele huwelijk ligt na de peildatum

Reply:
Ik vind de voorgestelde titel niet technisch...:-)

Reply:
> Nog een vraagje: De wijze waarop de titel is opgesteld is toch niet aan voorwaarden verbonden, of wel? Ik ben er tot nu toe tenminste vanuit gegaan dat de programmatuur de titel niet interpreteert.

Nee, er zijn geen voorwaarden verbonden aan de titel en programmatuur doet er niets mee.

Reply:
Ok, duidelijk.
Het scenario is inmiddels niet meer abstract.